File: 2_Periodo/AED_II/fonte_aula/MATRIZ_DINÂMICA/Celula.java
class Celula{
	int elemento;
	Celula prox;
	Celula ant;
	Celula sup;
	Celula inf;

	public Celula(int elemento){
		this.elemento = elemento;
		this.prox = null;
		this.ant = null;
		this.sup = null;
		this.inf = null;
	}// End Celula()

	public Celula(){
		this.elemento = 0;
		this.prox = null;
		this.ant = null;
		this.sup = null;		
	}// End Celula()
}// End class Celula

File: 2_Periodo/AED_II/fonte_aula/MATRIZ_DINÂMICA/Celula.c
#include <stdlib.h>
typedef struct Celula{
	int elemento;
	struct Celula* prox;
	struct Celula* ant;
    struct Celula* sup;
	struct Celula* inf;
}Celula;

void inicioCelula(int x){
	Celula* celula = (Celula*)malloc(sizeof(Celula));
	celula->elemento = x;
	celula->prox = NULL;
	celula->ant = NULL;
	celula->sup = NULL;
	celula->inf = NULL;
}// End startCelula()

void start(){
	Celula* celula = (Celula*)malloc(sizeof(Celula));
	celula->elemento = 0;
	celula->prox = NULL;
	celula->ant = NULL;
	celula->sup = NULL;
	celula->inf = NULL;
}// End startCelula() 
int main(void){
	return 0;
}